quicktill/pdrivers.py

```diff
@@ -701,6 +720,49 @@ def process_canvas(self, canvas, f):
                 f.write(escpos.ep_fullcut)
             f.flush()
 
+    def _image(self, data, width, height, f):
+        # Print a PBM bit-image
+        if width > self.dpl:
+            # Image too wide for paper
+            return
+
+        # Calculate padding required to center the image
+        padding = (self.dpl - width) // 2
+        padchars = [False] * padding
+
+        # Partition the bitmap into lines
+        lines = []
+        for line in range(height):
+            start = line * (width // 8)
+            end = start + (width // 8)
+            line = padchars.copy()
```

Comment:
Some unintented variable shadowing going on here (redeclaration of `line`).  Cleaning up this up during some refactoring to use `batched` again here.